src/components/ModiteList/index.tsx

```diff
@@ -104,7 +109,15 @@ const ModiteList: FunctionComponent<ModiteListProps> = ({
               lastScrollOffset={lastScrollOffsetModites}
             />
           )}
-          {!isDetails && !listRecords.length && <SkeletonList />}
+          {!isDetails && !listRecords.length && filter.length && (
+            <VirtualizedList
```

Comment:
This is great, thanks for submitting the fix.

Just an idea: how about creating a custom component for `NoRecordsFound`?

e.g.

```jsx
const NoRecordsFound = () => {
  const moditeNotFound = {
    real_name: 'No modite found',
    tz: (Intl.DateTimeFormat().resolvedOptions().timeZone) // client timezone
  }
  
  return  <VirtualizedList {{...allTheOtherStuffHere}} />
}
```

Reply:
Sure! Will do it.